explainer.md

```diff
@@ -827,6 +828,16 @@ facts were considered by our decision process.
   [WriteOptions.sync option](https://github.com/google/leveldb/blob/master/doc/index.md#synchronous-writes).
 
 
+### Default Quota
+
+We propose the default quota for a storage bucket to match the origin quota.
+This may seem unintuitive to have quota for a storage bucket be so
+large, disconnecting further from available disk space.
+However having anything under 100% of the origin quota will become a
```

Comment:
Let's add "We think" to this sentence and the next one, to soften them.

As far as I know, other browsers (especially Firefox) plan to take a different approach here. They currently have much smaller quotas (1-5GB / origin). I think their default bucket will match their current quotas, and they'll grant developers additional quota as they create additional buckets.

There is some merit to this approach -- the browser is in a slightly better position to handle storage pressure gracefully, as it's guaranteed to have O(number of GB used) buckets to choose from. We (Chrome) don't think this is the best approach for the ecosystem, so we won't be pursuing it. But we (the Buckets API designers) should come up with a design that allows this approach, unless we become convinced that it's completely wrong / harmful to the ecosystem.

Reply:
This makes sense, thanks for this explanation!